proposals/0425-decl-invis-binders.rst

```diff
@@ -621,6 +621,15 @@ The proposed changes provide the programmer with a more principled way of
 brining type variables into scope in certain corner cases, simplify arity
 inference and scoping rules.
 
+This gets us closer to upholding two principles:
+
+- The `Explicit Binding Principle`_, because the new ``@``-binders is explicit unlike previous alternatives
+
+- The `Lexical Scoping Principle`_, because the new simplified rules about variables needing to occur on the LHS in certain situations.
```

Comment:
```suggestion
- The `Lexical Scoping Principle`_, because of the new simplified rules about variables needing to occur on the LHS in certain situations.
```